_episodes/04-units.md

```diff
@@ -24,14 +30,15 @@ project and language to language.  A good guideline is that if the code cannot
 be made any simpler logically (you cannot split apart the addition operator) or
 practically (a function is self-contained and well defined), then it is a unit. 
 
-> ## Functions are Like Paragraphs {.callout}
+> ## Note to self
```

Comment:
Why is this "Note to Self" ? I don't get it. It should be called "Functions are Like Paragraphs".